### PR TITLE
Feat/portfolio coin select

### PR DESCRIPTION
**Pull request by jvin08**

### CODE REVIEW

File: app/components/portfolio/CoinSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      coin: selectedCoin[0],
+      symbol: selectedCoin[1],
+      amount: amount,
+      purchaseTime: time,
+      image: coinImage,
+    };
+    if(activeSaveBtn){
+      const currentStorage = JSON.parse(localStorage.getItem("coins") || "[]");
+      const updatedStorage = [...currentStorage, storageObject];
+      localStorage.setItem("coins", JSON.stringify(updatedStorage));
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/CoinSelect.tsx:48]: maybe you could implement a useLocalState custom hook

File: app/components/portfolio/SearchResults.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      if (ref.current && !ref.current.contains(event.target as Node)) {
+        toggleSearch();
+        clearSearch();
+      }
+    };
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  },[clearSearch, toggleSearch]);
```
Comment (yoloOnTheBattlefield) [inline app/components/portfolio/SearchResults.tsx:50]: I would extract this clickoutside logic and put it into a custom hook in the next PR

**Review by yoloOnTheBattlefield — APPROVED** [review]

File: app/components/portfolio/SearchResults.tsx (hunk 2 above)
Comment (jvin08) [reply app/components/portfolio/SearchResults.tsx:50]: I agree with suggestion

File: app/components/portfolio/CoinSelect.tsx (hunk 1 above)
Comment (jvin08) [reply app/components/portfolio/CoinSelect.tsx:48]: I agree with suggestion

**Pull request merged**